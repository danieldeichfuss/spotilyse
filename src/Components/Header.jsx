import React from "react";

function Header() {
  return (
    <header className="Header">
      <h1 className="Header__title">Spotilyse</h1>
      <p className="Header__intro">Your Spotify personality dashboard</p>
    </header>
  );
}

export default Header;
