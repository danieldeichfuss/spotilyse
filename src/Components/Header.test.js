import React from "react";
import Header from "./Header";
import { shallow, configure } from "enzyme";
import ReactSixteenAdapter from "enzyme-adapter-react-16";
import { shallowToJson } from "enzyme-to-json";

configure({ adapter: new ReactSixteenAdapter() });

const wrapper = shallow(<Header />);

it("renders page correctly", () => {
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
