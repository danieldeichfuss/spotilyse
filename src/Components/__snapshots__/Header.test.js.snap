// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page correctly 1`] = `
<header
  className="Header"
>
  <h1
    className="Header__title"
  >
    Spotilyse
  </h1>
  <p
    className="Header__intro"
  >
    Your Spotify personality dashboard
  </p>
</header>
`;
